src/snowflake/connector/pandas_tools.py

```diff
@@ -165,7 +165,7 @@ def write_pandas(conn: 'SnowflakeConnection',
             copy_results)
 
 
-def pd_writer(table: pandas.io.sql.SQLTable,
+def pd_writer(table: 'pandas.io.sql.SQLTable',
```

Comment:
Whats the reason for this change?

Reply:
Since I'm switching to tagging test, instead of ignoring a growing list of sub-directories we need to be more careful on how we add Type hints.
If we didn't put this type into the string then Python would need to resolve `pandas.io.sql.SQLTable` (in which case our `options` sub-package would throw an Exception when the pandas optional dependency is not installed).
For this exact reason Python allows the types to be put into a string, so PyCharm and `mypy` should have no problem with the string.